src/store/modules/accounts/accounts.ts

```diff
@@ -168,9 +168,17 @@ const accounts_module: Module<AccountsState, RootState> = {
         },
 
         async updateKycStatus({ state, rootState }) {
-            const wallet = rootState.activeWallet
-            if (!wallet) return
-            state.kycStatus = await checkVerificationStatus('0x' + wallet.ethAddress)
+            if (!rootState.activeWallet || rootState.activeWallet.type === 'ledger') return null
+            const wallet = rootState.activeWallet as SingletonWallet | MnemonicWallet
+            try {
+                state.kycStatus = await checkVerificationStatus(
+                    wallet.ethKey,
+                    //@ts-ignore
```

Comment:
Why is ts-ignore here? Can't we do it properly without ignoring?

Reply:
cant because the network is not defined yet